ui/src/alerting/components/CheckCard.tsx

```diff
@@ -170,7 +171,13 @@ const CheckCard: FunctionComponent<Props> = ({
         />
       }
       metaData={[
+        <>Last completed at {check.latestCompleted}</>,
         <>{relativeTimestampFormatter(check.updatedAt, 'Last updated ')}</>,
+        <LastRunTaskStatus
+          key={2}
```

Comment:
added the `key` here to resolve linter errors

Reply:
is this a react warning or a lint error? if react is generating a warning then that's the issue we should address. if lint is mistakenly saying we need a key, we should comment an override here, because it's adding meaning to the code to get around a tool issue. do you understand the difference? a real react warning about a list item not having a key is something we should fix, a mistaken lint error trying to head off that warning is something we should comment about, but not change our code to make go away.

if it's an actual react warning, we should give this a more unique name, if possible. react says to only use the item's list index as a last resort

>When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort:

>We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. 

https://reactjs.org/docs/lists-and-keys.html#keys

same comment and any changes that need to happen apply to `RuleCard`'s `key` change

Reply:
It was a lint error - i imagine stemming from the way `clockface` handles metaData being passed dom nodes